schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -247,54 +258,72 @@
-                                    ">=",
-                                    "!",
-                                    "!<",
-                                    "!>",
-                                    "!<=",
-                                    "!>=",
-                                    "?",
-                                    "?<",
-                                    "?>",
-                                    "?<=",
-                                    "?>="
-                                ]
+                            "limit": {
+                                "description": "The upper limit of the range. This is the least version not in the range (that is, a non-inclusive upper limit). The usual version syntax is expanded to allow a pattern to end in an asterisk (*), indicating an arbitrarily large number in the version ordering. For example, {version: 1.0 limit: 1.*} would describe the entire 1.X branch for most range kinds, and {version: 2.0, limit: *} describes all versions starting at 2.0, including 3.0, 5.1, and so on.",
+                                "$ref": "#/definitions/version"
+                            },
+                            "repo": {
+                                "description": "The URL of the source code repository, for use with 'range' set to 'git'.",
+                                "$ref": "#/definitions/uriType"
                             },
-                            "references": {
-                                "$ref": "#/definitions/references"
+                            "changes": {
+                                "type": "array",
+                                "description": "A list of status changes that take place during the range. The array should be sorted by the 'at' field, according to the range kind, but clients must re-sort the list themselves rather than assume it is sorted.",
+                                "minItems": 1,
+                                "uniqueItems": true,
+                                "items": {
+                                    "type": "object",
+                                    "description": "The start of a single status change during the range.",
+                                    "required": ["at", "status"],
+                                    "properties": {
+                                        "at": {
+                                            "description": "The version at which a status change occurs.",
+                                            "$ref": "#/definitions/version"
+                                        },
+                                        "status": {
+                                            "description": "The new status in the range starting at the given version.",
+                                            "$ref": "#/definitions/status"
+                                        }
+                                    },
+                                    "additionalProperties": false
```

Comment:
restricting additionalProperties seems redundant here since this is declared at a higher level that allows experimental fields with x_fieldName.
We should allow such experimental fields (for factoring in 'wontfix', 'unsupported', 'confirmed-with-test', etc., in the future).

Reply:
Removed. I don't know what the conventions are here, I just noticed that some objects had them, so I thought all were supposed to.

Reply:
What about the version and product objects? Should they allow additional fields?

Reply:
Deleted the other two as well.